src/server/server.js

```diff
@@ -78,6 +78,15 @@ if (env === 'development') {
     );
 }
 
+if (process.env.NODE_ENV === 'production') {
+    const resolvedAssets = require(path.join(
```

Comment:
define resolvedAssets outside of this conditional block so that it's false rather than undefined in the surrounding block where it's used later, to reduce ambiguity